meson.build

```diff
@@ -25,6 +25,7 @@ executable(
     dependency('glib-2.0'),
     dependency('granite'),
     dependency('gtk+-3.0'),
+    dependency('libhandy-1', version: '>=1.0.0'),
```

Comment:
Don't forget to also update `deb-packaging` branch once you are done and add `libhandy-1-dev (>= 1.0.0),` to `debian/control`.